_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Deploy Forseti Security on Google Kubernetes Engine
+order: 004
+---
+
+# {{ page.title }}
```

Comment:
Is this a remnant that should be updated?

Reply:
@RexBelli thanks for the review.  Can you please clarify?

Reply:
Sure, I viewed the page directly and saw `{{ page.title }}` remained: https://github.com/kevensen/forseti-security/blob/forsetisecurity.org-dev/_docs/_latest/setup/forseti-on-gke.md

Reply:
gotcha.  May I recommend you build the website locally?

https://forsetisecurity.org/docs/latest/develop/website.html

Reply:
Ah!  I wasn't sure if this might be a template for something else, thanks for the clarification!

Reply:
not a problem.  Thanks for taking a look at the docs.